zilencer/management/commands/populate_db.py

```diff
@@ -318,6 +318,14 @@ def handle(self, **options):
             ]
             create_users(zulip_realm, zulip_cross_realm_bots, bot_type=UserProfile.DEFAULT_BOT)
 
+            zulip_outgoing_bots = [("Outgoing Webhook", "outgoing-webhook@zulip.com")]
+            create_users(zulip_realm, zulip_outgoing_bots, bot_type=UserProfile.OUTGOING_WEBHOOK_BOT)
+            Service.objects.create(name="test",
+                                   user_profile=get_user_profile_by_email("outgoing-webhook@zulip.com"),
+                                   base_url="http://0.0.0.0:5002/bots/followup",
```

Comment:
Why is this `0.0.0.0:5002`?  Seems like maybe we want it to be `127.0.0.1:5002` instead.